src/Hosting/Server.IntegrationTesting/src/Deployers/SelfHostDeployer.cs

```diff
@@ -157,6 +171,11 @@ protected async Task<(Uri url, CancellationToken hostExitToken)> StartSelfHostAs
                         {
                             actualUrl = new Uri(m.Groups["url"].Value);
                         }
+
+                        if (dataArgs.Data.Contains("HttpSysException: The process cannot access the file because it is being used by another process."))
```

Comment:
Better. Does it work? The app exits pretty quickly after this gets thrown and you might not get a chance to capture this.

Reply:
Reading documentation now. It's not well defined.

Reply:
I don't believe this is reliable. Thinking of other solutions.